vuepress/docs/next/tutorials/consume/mt-redis.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebarDepth: 2
```

Comment:
Redis is useful for High Availability, not just MT, so I'd recommend changing the name of the file.